faqs/galaxy/histories_list.md

```diff
@@ -0,0 +1,14 @@
+---
+title: View a list of all histories
+description: This FAQ demonstrates how to list all histories for a given user
+area: histories
+box_type: tip
+layout: faq
+contributors: [nekrut]
+---
+
+Click the **Switch history** icon at the top of the history panel to bring up a list of all your histories:
+
+![UI for creating new history]({% link shared/images/histories_list.svg %})
+
```

Comment:
given that there are like, three ways to do this now, should we also include those others and maybe compare them a bit?

there's:

- histories (activity bar) (I would honestly recommend making this the first suggestion for "list all of your histories" rather than the switcher.)
- "Switch to history"  from the quick switcher (doesn't provide the nicer view/filtering, but does let you open them in the multi-view
- dropdown → "show histories side by side"

and then comment on the reasons you'd use each?

Reply:
awesome, thanks so much! :)